src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -63,7 +63,6 @@
 					@update:model-value="toggleConversationsListStyle" />
 				<!-- FIXME: remove v-if after implementing split view -->
 				<NcFormBoxSwitch
-					v-if="false"
```

Comment:
Now thinking, this should be hidden for Talk < 23 | Nextcloud < 33. Request to server will fail, and user will be locked in split view 
```vue
v-if="supportSplitView"
```